README.md

```diff
@@ -743,9 +742,23 @@ class ArticlesController < ApplicationController
   end
 end
 ```
+
+One can write `ActionDispatch::Integration` test helpers for `login` and
+`logout` by making requests to the rodauth endpoints
+
 ```rb
 # test/controllers/articles_controller_test.rb
 class ArticlesControllerTest < ActionController::TestCase
```

Comment:
You probably meant to inherit form `ActionDispatch::IntegrationTest` here 😉

Reply:
💯 how embarrassing!

Reply:
It needs to be `ActionDispatch::IntegrationTest`, not `ActionDispatch::Integration` (the latter is a module).

Reply:
Holy moly. I really appreciate that you are being thorough.

Me:

![sideshow_bob_rakes](https://user-images.githubusercontent.com/97257/203553043-983e9bcd-b33f-40e3-9fb1-635fe6f36802.gif)